benchmark/child_process/child-process-read.js

```diff
@@ -29,7 +29,12 @@ function main(conf) {
   });
 
   setTimeout(function() {
-    child.kill();
+    if (process.platform === 'win32') {
+      // Sometimes there's a yes.exe process left hanging around on Windows...
+      child_process.execSync(`taskkill /f /t /pid ${child.pid}`);
```

Comment:
CI is failing...

>     ReferenceError: child_process is not defined

Only the `child_process.spawn` is stored in a variable above..